charts/argo-workflows/templates/controller/workflow-controller-config-map.yaml

```diff
@@ -131,19 +131,18 @@ data:
         name: {{ .clientSecret.name }}
         key: {{ .clientSecret.key }}
       redirectUrl: {{ .redirectUrl }}
-      {{- if and (.rbac) (.rbac.enabled) }}
+      {{- with .rbac }}
```

Comment:
Helm evaluate both of `.sessionExpiry` and `.sessionExpiry.duration` until helm v3.9.x, even if `.sessionExpiry` returns false.
So I switched to use `with`.